indexer/src/main/java/au/org/aodn/esindexer/service/GeoNetworkServiceImpl.java

```diff
@@ -420,8 +415,8 @@ public boolean isMetadataRecordsCountLessThan(int c) {
             backoff = @Backoff(delay = 1500L)
     )
     @Override
-    public Iterable<String> getAllMetadataRecords() {
-        SearchRequest req = createSearchAllUUID(startingUUID);
+    public Iterable<String> getAllMetadataRecords(String beginWithUUid) {
```

Comment:
I think this can be named something related to "partition" or "segment" as it is not really "all" records

Reply:
StartingFrom etc.